ts-client/src/dlmm/index.ts

```diff
@@ -4303,6 +4279,39 @@ export class DLMM {
     }).add(syncWithMarketPriceTx);
   }
 
+  public async getMaxPrice(
+    minPrice: string,
+    binArrays?: BinArrayAccount[]
+  ): Promise<string> {
+    const _binArrays = binArrays || (await this.getBinArrays());
```

Comment:
Why use `_binArrays `? I guessed _variable is used for ignore variable?

Reply:
Do you fetch all binArrays? caused that may crash client, we should only read the binArray having liquidity and it's bin id is the biggest.
That can be archived through bitmap

Reply:
> Do you fetch all binArrays? caused that may crash client, we should only read the binArray having liquidity and it's bin id is the biggest. That can be archived through bitmap

wdym? fetching all binArrays just using 1 gPA?